libs/sdk/lib/JsonLinkedDataProofs.ts

```diff
@@ -0,0 +1,177 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Crypto, JoseBuilder, CryptoHelpers } from './index';
+import { IPayloadProtectionSigning } from 'verifiablecredentials-crypto-sdk-typescript-protocols-common';
+import { PublicKey } from 'verifiablecredentials-crypto-sdk-typescript-keys';
+import base64url from 'base64url';
+import { CryptoAlgorithm } from 'verifiablecredentials-crypto-sdk-typescript-keystore';
+import { SubtleCryptoExtension } from 'verifiablecredentials-crypto-sdk-typescript-plugin';
+const jsonld = require('jsonld');
+const clone = require('clone');
+
+export default class JsonLinkedDataProofs {
+
+  /**
+   * Create instance of <see @class Jose>
+   * @param builder The builder object
+   */
+  constructor(
+    private _signer: IPayloadProtectionSigning) {
+  }
+
+  private _credential: any | undefined;
+
+  /**
+   * Embed the signature into the payload
+   * @param payload to embed signature
+   */
+  public async sign(payload: object): Promise<JsonLinkedDataProofs> {
+
+    if (!payload) {
+      throw new Error('JSON LD proofs input is undefined');
+    }
+
+    if (typeof payload !== 'object') {
+      throw new Error('JSON LD proofs input should be an object');
+    }
+    const crypto = this._signer.builder.crypto;
+    const [verifyData, proof] = await this.createVerifyData(payload, crypto);
+
+    // sign payload
+
+
+    let jwsSigner: IPayloadProtectionSigning = new JoseBuilder(this._signer.builder.crypto)
+      .build();
+
+    jwsSigner = await jwsSigner.sign(verifyData);
+    const jws = jwsSigner.serialize();
+    const splitter = jws.split('.');
+    // Set payload
+    const payloadToSign = Buffer.concat([Buffer.from(splitter[0]), Buffer.from('.'), verifyData]);
+    
+    const alg = 'eddsa';
```

Comment:
```suggestion
    const alg = 'EdDSA';
```
https://www.iana.org/assignments/jose/jose.xhtml